source/code-generation/query-converter/manage-queries.txt

```diff
@@ -0,0 +1,41 @@
+.. _rm-manage-queries:
+
+==============
+Manage Queries
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator provides various methods for managing your 
+queries while converting and testing.
+
+The following actions are available:
```

Comment:
(question)
Do we need to describe where the actions are and how to ... use them?